packages/manager/apps/hub/src/dashboard/translations/Messages_de_DE.json

```diff
@@ -1,6 +1,5 @@
 {
-  "manager_hub_dashboard_welcome": "Herzlich Willkommen, {{ name }} !",
```

Comment:
Moved in https://github.com/ovh/manager/commit/4c8c67e56e8c13b4d0e53fe26afc5425c6a4223f#diff-843b563295745bc8de404ac7fc10be76